.acquia/pipeline.yaml

```diff
@@ -0,0 +1,76 @@
+---
+type: default
+team: ngc
+service: fn-kubexit
+
+# Validate the formatting of the pipeline.yaml file.
+validate_config: true
+
+environment_image:
+  file: ".acquia/Dockerfile.ci"
+  context: "."
+  build_args:
+    - secrets:
+        - type: vault
+          key: SSH_KEY
+          value: GIT_SSH_KEY
+          path: secret/pipeline-default/GIT_SSH_KEY
+
+pre_build:
+  security_composition_analysis:
+    required: false
+    language:
+    - go
+  clone_source_image:
+  - steps:
+    - | 
+      export KUBEXIT_VERSION=v0.3.2-acquia7
+      git clone --depth 1 --branch "${KUBEXIT_VERSION}" https://github.com/karlkfi/kubexit.git mirror
```

Comment:
Why we are cloning/using `karlkfi/kubexit`? I believe `acquia/kubexit` has additions (see history).
I see rest of the files (source) has been deleted.

Reply:
Yes, I believe, we need to clone the actual repo and apply the history (only bug fix - tombstone one), as git patch.
Just like how we did for other repo, fn-apache-exporter, fn-memcached-exporter etc.

cc @alexxed

Reply:
we're changing this to not be a fork, but to work with our regular mirror process:
- clone from source
- apply any patches
- build image

Reply:
@renishb10  
Where are the step(s) for `apply the history (only bug fix - tombstone one), as git patch`? 
I'm not sure how we will add fixes or feature / update go.mod in `acquia/kubexit` in future.

Reply:
in separate patches files, e.g. patch1, patch2 in the root dir of the repo

Reply:
I've added a directory named `/patches` and added wildcard to the git apply cmd
```
git apply --reject --whitespace=fix ../patches/*.patch
```